stories/Icon/Icon.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Icon, Text, icons } from 'components';
+import { ComponentMeta } from '@storybook/react';
+import docs from './readme.md';
+
+export default {
+  title: 'Example/Icons',
+  component: Icon,
+  parameters: {
+    docs: {
+      description: {
+        component: docs,
+      },
+    },
+  },
+  argTypes: {
+    backgroundColor: { control: 'color' },
+  },
+} as ComponentMeta<typeof Icon>;
+
+export const Icons = () => {
+  return (
+    <div style={{ display: 'flex', flexWrap: 'wrap' }}>
```

Comment:
1. Add search input for icons name
2. Separate icons by size blocks
3. Change icon name text color to gray